packages/lodestar-config/mainnet.d.ts

```diff
@@ -0,0 +1 @@
+export * from "./lib/presets/mainnet";
```

Comment:
can we have `lodestar-config/mainnet.ts` instead?

Reply:
That would require compiling code outside of source, I haven't found a clean way to do it